lib/common/options.c

```diff
@@ -584,20 +585,40 @@ pcmk__format_option_metadata(const char *name, const char *desc_short,
             continue;
         }
 
+        g_string_append_printf(s, "    <parameter name=\"%s\">\n",
+                                  option_list[lpc].name);
+
         escaped_long = crm_xml_escape(option_list[lpc].description_long?
-                                         _(option_list[lpc].description_long) :
-                                          _(option_list[lpc].description_short));
-        escaped_short = crm_xml_escape(_(option_list[lpc].description_short));
+                                         option_list[lpc].description_long :
+                                          option_list[lpc].description_short);
+        escaped_short = crm_xml_escape(option_list[lpc].description_short);
 
-        g_string_append_printf(s, "    <parameter name=\"%s\">\n"
+        g_string_append_printf(s,
                                   "      <longdesc lang=\"en\">%s%s%s</longdesc>\n"
                                   "      <shortdesc lang=\"en\">%s</shortdesc>\n",
-                                  option_list[lpc].name,
                                   escaped_long,
                                   (option_list[lpc].values? "  Allowed values: " : ""),
                                   (option_list[lpc].values? option_list[lpc].values : ""),
                                   escaped_short);
-        free(escaped_long);
```

Comment:
escaped_long and escaped_short do still need to be freed before reusing them for NLS (though it's probably better to use different variables for NLS, so we can compare them to avoid duplicates)